Themes/default/languages/Install.english.php

```diff
@@ -231,10 +262,70 @@
+$txt['upgrade_bypass'] = '<strong>Note:</strong> If necessary the above security check can be bypassed for users who may administrate a server but not have admin rights on the forum. In order to bypass the above check simply open &quot;upgrade.php&quot; in a text editor and replace &quot;$disable_security = false;&quot; with &quot;$disable_security = true;&quot; and refresh this page.';
+$txt['upgrade_areyouready'] = 'Before the upgrade gets underway please review the options below - and hit continue when you\'re ready to begin.';
+$txt['upgrade_backup_table'] = 'Backup tables in your database with the prefix';
+$txt['upgrade_recommended'] = 'recommended!';
+$txt['upgrade_maintenace'] = 'Put the forum into maintenance mode during upgrade.';
+$txt['upgrade_maintenance_title'] = 'Maintenace Title:';
+$txt['upgrade_maintenace_message'] = 'Maintenance Message:';
+$txt['upgrade_customize'] = 'Customize';
+$txt['upgrade_debug_info'] = 'Output extra debugging information';
+$txt['upgrade_empty_errlog'] = 'Empty error log before upgrading';
+$txt['upgrade_delete_karma'] = 'Delete all karma settings and info from the DB';
+$txt['upgrade_stats_collection'] = 'Allow Simple Machines to Collect Basic Stats Monthly.';
+$txt['upgrade_stats_info'] = 'If enabled, this will allow Simple Machines to visit your site once a month to collect basic statistics. This will help us make decisions as to which configurations to optimise the software for. For more information please visit our <a href="https://www.simplemachines.org/about/stats.php" target="_blank" rel="noopener">info page';
+$txt['upgrade_db_changes'] = 'Executing database changes';
+$txt['upgrade_db_patient'] = 'Please be patient - this may take some time on large forums. The time elapsed increments from the server to show progress is being made!';
+$txt['upgrade_db_complete'] = '1 Database Updates Complete! Click Continue to Proceed.';
+$txt['upgrade_db_complete2'] = 'Database Updates Complete! Click Continue to Proceed.';
+$txt['upgrade_script'] = 'Executing upgrade script';
+$txt['upgrade_error'] = 'Error!';
+$txt['upgrade_unknown_error'] = 'Unknown Error!';
+/* Same sentence, 3 different strings */
+$txt['upgrade_completed'] = 'Completed';
+$txt['upgrade_outof'] = 'out of';
+$txt['upgrade_tables'] = 'tables.';
+
+$txt['upgrade_completed_table'] = 'Completed Table:';
+$txt['upgrade_current_table'] = 'Current Table:';
+$txt['upgrade_fulltext'] = 'Please note that your fulltext index was dropped to facilitate the conversion and will need to be recreated in the admin area after the upgrade is complete.';
+$txt['upgrade_conversion_proceed'] = 'Conversion Complete! Click Continue to Proceed.';
+$txt['upgrade_convert_datajson'] = 'Converting data from serialize to JSON...';
+$txt['upgrade_json_completed'] = 'Convert to JSON Complete! Click Continue to Proceed.';
+$txt['upgrade_executing'] = 'Executing:';
+$txt['upgrade_of'] = 'of';
+$txt['upgrade_admin_login'] = 'Admin Login:'
```

Comment:
missing semi-colon.

Reply:
>missing semi-colon.
This wasnot the bug for correct loading language file. In my translate was the semi-colon.